templates/d/dub.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "cart",
+  "targetType": "executable",
+  "configurations": [
+    {
+      "name": "raw",
+      "dflags": [
+        "-betterC"
+      ],
+      "lflags": [
+        "--strip-all",
+        "--allow-undefined",
+        "--stack-first",
+        "--no-entry"
```

Comment:
We also need to pull over:

 ```
        "--initial-memory=65536",
        "--max-memory=65536",
        "-zstack-size=14752"
```

But with TIC-80 specific values, which I think would mean 256kb memory and 96kb "reserved"... which I THINK means stack-size should be `96*1024 + 8 * 1024` (assuming we want an 8kb actual stack)... does that sound right?

Reply:
Let me run some tests with the numbers.

Reply:
So the numbers for TIC-80 would be, for 8k actual stack:
```
       "--initial-memory=262144",
       "--max-memory=262144",
       "-zstack-size=106496"
```

Reply:
Did we ever get this changed?